pkg/features/versioned_kube_features.go

```diff
@@ -239,7 +239,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	},
 
 	genericfeatures.APIResponseCompression: {
-		{Version: version.MustParse("1.8"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
The feature-gate was introduced in v1.8.0, not v1.7.0 even though the milestone of that PR is 1.7.

Please check the following links:
- https://github.com/kubernetes/kubernetes/blob/release-1.8/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L50 
- https://github.com/kubernetes/kubernetes/blob/release-1.7/staging/src/k8s.io/apiserver/pkg/features/kube_features.go (no feature-gate)

I'm unsure if that PR was reverted.

Reply:
Good Catch, and the PR milestone is wrong.